icubam/backoffice/handlers/regions.py

```diff
@@ -0,0 +1,53 @@
+"""Creating/edition of Regions."""
+import tornado.escape
+import tornado.web
+
+from icubam.backoffice.handlers import base
+from icubam.backoffice.handlers import home
+from icubam.db import store
+
+
+class ListRegionsHandler(base.BaseHandler):
+  ROUTE = "/list_regions"
+
+  def prepare_for_table(self, region):
+    result = region.to_dict(max_depth=0)
+    for key in ['last_modified', 'icu']:
+      result.pop(key, None)
+    return self.format_list_item(result)
+
+  @tornado.web.authenticated
+  def get(self):
+    if self.user.is_admin:
```

Comment:
Isn't there a decorator to make endpoints only accessible by a given user? 

Otherwise we will write this often, which is also fine, just a comment.

Reply:
Very good idea. I have added an AdminHandler now